MacrobenchmarkSample/build.gradle

```
@@ -17,7 +17,7 @@
 // Top-level build file where you can add configuration options common to all sub-projects/modules.
 buildscript {
     ext {
-        agpVersion = '7.3.0-beta01'
+        agpVersion = '7.4.0-alpha03'
```

```suggestion
        agpVersion = '7.3.0-beta05'
```
I don't think we want to update to alpha. Once Dolphin is stable and baseline profiles don't require alpha/beta AGP, we should keep it stable. Otherwise we force folks to use Android Studio canary